### DISCUSSION TITLE
<title>Suspended on school account. Need this account

### DISCUSSION THREAD

**Discussion by kreyeshuynh5** [community/community#53507] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I was suspended on my school account I don’t know why. I need this account to push projects for school. My ticket number is: 2121930. I would hope this issue gets solved quickly because I have school work to push. Could someone help me get in contact directly with github support.

**Comment by Truonghau83** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> I was suspended on my school account I don’t know why. I need this account to push projects for school. My ticket number is: 2121930. I would hope this issue gets solved quickly because I have school work to push. Could someone help me get in contact directly with github support.
